lib/config/ludwig.js

```diff
@@ -90,7 +94,11 @@ module.exports = [
         'description':'L\'aide au logement est une aide financière destinée à réduire le montant de votre loyer ou votre mensualité d’emprunt immobilier. Elle est attribuée selon la nature de votre logement et la composition de votre famille.',
         'imgSrc':'logo_caf.png',
         'link':'https://www.caf.fr/aides-et-services/s-informer-sur-les-aides/logement-et-cadre-de-vie/les-aides-au-logement-0',
-        'isBaseRessourcesYearMoins2':true
+        'isBaseRessourcesYearMoins2':true,
+        'uncomputabilityReasons': {
+            'primo_accedant': 'le demandeur est primo-accédant',
+            'locataire_foyer': 'le demandeur loge dans un foyer'
+        }
```

Comment:
Seems like we are de-synchronizing [droits.js](https://github.com/sgmap/mes-aides-ui/blob/uncomputability-ludwig/app/js/constants/droits.js) and  [ludwig.js](https://github.com/sgmap/mes-aides-api/blob/uncomputability-ludwig/lib/config/ludwig.js)